### PR TITLE
ci: Switch to GHA

### PR DESCRIPTION
**Pull request by dewyatt**
TODO:
* ~~Need a token for codecov - @ronaldtse can you create this in a secret env var?~~
* ~~I had to add a hack in 838a65f, so I want to track down the underlying issue there before review~~ (mysteriously resolved itself?)

### CODE REVIEW

**Comment by dewyatt** [comment]
I meant to do a draft PR here btw, oops! Oh well, no way to convert it apparently.

**Comment by ronaldtse** [comment]
@dewyatt If you go to codecov.io you should be able to login to the rnpgp account (https://codecov.io/gh/rnpgp).

On the other hand, the settings page says this:

Screen Shot 2019-11-07 at 7 16 16 PM

Do we really need the codecov token to be set as a secret?

File: ci/install.sh [hunk 1]
```diff
@@ -117,130 +49,72 @@ build_gpg_beta() {
...
-    --with-ksba-prefix="${GPG_INSTALL}" \
-    --with-npth-prefix="${GPG_INSTALL}" \
-    --disable-ldap \
-    --disable-doc \
-    --enable-maintainer-mode
-  ${MAKE} -j${MAKE_PARALLEL} install
+  gpg_build="$PWD"
+  gpg_install="$GPG_INSTALL"
+  mkdir -p "$gpg_build" "${gpg_install}"
+  git clone --depth 1 https://github.com/riboseinc/gpg-build-scripts
```
Comment (ronaldtse) [inline ci/install.sh]: @dewyatt should we move this to the rnpgp org?

**Comment by ronaldtse** [comment]
No worries, we all know it's not to be merged until all the lights turn green 😉

File: ci/install.sh (hunk 1 above)
Comment (dewyatt) [reply ci/install.sh]: @ronaldtse Probably, it certainly wouldn't hurt.

Comment (ronaldtse) [reply ci/install.sh]: Transferred.

**Comment by ronaldtse** [comment]
@dewyatt this is all passing! Are we ready to merge?

**Comment by dewyatt** [comment]
> @dewyatt this is all passing! Are we ready to merge?

Not yet

**Comment by dewyatt** [comment]
Should be good now I believe.

**Comment by ronaldtse** [comment]
Just CI so merging. 

@dewyatt can you remove the Travis config and stop it from building there? Thanks!

**Pull request merged**